frontend/src/components/design-system/input/ButtonComponent.vue

```diff
@@ -0,0 +1,267 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+
+import ButtonText from '@/components/design-system/typography/ButtonText.vue';
+
+export interface Props {
+  tag?: 'button' | 'a';
+  href?: string;
+  kind: 'link' | 'outlined' | 'filled' | 'answer';
+  size?: 'medium' | 'small';
+  color?: 'neutral' | 'primary' | 'secondary';
+  selected?: boolean;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  tag: 'button',
+  href: undefined,
```

Comment:
pokud nekdo zada href mel by se root komponent automaticky zmenit na `a`

Reply:
Jo, na to jsem zapomněl, přidáno.